src/composables/percentageTool.ts

```diff
@@ -74,8 +81,8 @@ export function usePercentageTool(chartRef, theme: string) {
           style: {
             x: x,
             y: y - 20,
-            text: (startPrice < endPrice ? pct : '-' + pct) + '%',
-            font: '16px sans-serif',
+            text: `${timeDiff} Candles -  ${startPrice < endPrice ? pct : '-' + pct}%`,
```

Comment:
Haha, it's like you read my mind, was also going to add candle count. But I'd maybe call them bars? Just to be more compact.

Reply:
oh for sure, i'm not tied to any name, and i just added this rather quickly as a POC (it probably should wrap line, too).

i do like the way binance does this - with "4 bars - 4h" (on 1h timeframe).